packages/table/src/headers/columnHeader.tsx

```diff
@@ -60,6 +65,12 @@ export interface IColumnHeaderProps extends ISelectableProps, IColumnIndices, IC
     viewportRect: Rect;
 
     /**
+     * Enables/disables the reordering interaction.
+     * @default true
+     */
+    isReorderable?: boolean;
```

Comment:
is it reasonable to prevent specific columns from being reordered? like, does it make sense to support one-off disabling via this prop on `Column`, or is it more logical to prevent _all_ columns from being reordered via a single prop at a higher level?